SLAM_example.cpp

```
@@ -88,15 +88,15 @@ int main(int argc, char *argv[]) {
     auto status = core->startScanning();
     if(status != Status::SUCCESS) {
       std::cerr << "ERROR starting scanner" << std::endl;
-      return (int) status;
+      abort();
```

just curios, why abort instead of return status?